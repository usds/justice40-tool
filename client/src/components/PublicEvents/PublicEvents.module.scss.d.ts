declare namespace PublicEventsNamespace {
    export interface IPublicEventsScss {
        collectionContainer: string;
    }
  }

declare const PublicEventsScssModule: PublicEventsNamespace.IPublicEventsScss & {
    /** WARNING: Only available when `css-loader` is used without `style-loader` or `mini-css-extract-plugin` */
    locals: PublicEventsNamespace.IPublicEventsScss;
  };

  export = PublicEventsScssModule;
