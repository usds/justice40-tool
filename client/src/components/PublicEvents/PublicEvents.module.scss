@use '../../styles/design-system.scss' as *;

.collectionContainer{
    margin-bottom: 2rem !important;

}