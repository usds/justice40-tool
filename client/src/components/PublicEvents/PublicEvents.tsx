import React from 'react';
import {Collection} from '@trussworks/react-uswds';
import PublicEvent from '../PublicEvent';

import * as PUBLIC_ENG_COPY from '../../data/copy/publicEngage';
import * as styles from './PublicEvents.module.scss';

const PublicEvents = () => {
  return (
    <Collection className={styles.collectionContainer}>
      {PUBLIC_ENG_COPY.EVENTS.map((event, index) => <PublicEvent key={index} event={event} />)}
    </Collection>
  );
};

export default PublicEvents;
