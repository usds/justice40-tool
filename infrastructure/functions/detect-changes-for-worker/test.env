REGION=us-east-1
STAGE=sit

# ESC Cluster name that will from the containers
ECS_CLUSTER=j40-sit-justice40-data-harvester-ECSCluster-ktXGGU9zjwkb

# VPC Private Subnet that has a NAT GAteway
VPC_SUBNET_ID=subnet-07e68cb57322f7b1f

# Names of the container and task names.
GDAL_TASK_DEFINITION=sit-justice40-data-harvester-gdal
GDAL_CONTAINER_DEFINITION=sit-justice40-data-harvester-osgeo-gdal
TIPPECANOE_TASK_DEFINITION=sit-justice40-data-harvester-tippecanoe
TIPPECANOE_CONTAINER_DEFINITION=sit-justice40-data-harvester-mgiddens-tippecanoe

# AWS Credentials
AWS_ACCESS_KEY_ID=
AWS_SECRET_ACCESS_KEY=
AWS_DEFAULT_REGION=us-east-1