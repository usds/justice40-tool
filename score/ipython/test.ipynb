{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "1a4c0c68",
   "metadata": {},
   "outputs": [],
   "source": [
    "import numpy as np\n",
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "id": "70b3a793",
   "metadata": {},
   "outputs": [],
   "source": [
    "df = pd.read_csv('data/fips_states.csv')  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "id": "c514aad8",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>fips</th>\n",
       "      <th>state_name</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>Alabama</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>2</td>\n",
       "      <td>Alaska</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>4</td>\n",
       "      <td>Arizona</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>5</td>\n",
       "      <td>Arkansas</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>6</td>\n",
       "      <td>California</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   fips  state_name\n",
       "0     1    Alabama \n",
       "1     2     Alaska \n",
       "2     4    Arizona \n",
       "3     5   Arkansas \n",
       "4     6  California"
      ]
     },
     "execution_count": 51,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b9ee44d9",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
