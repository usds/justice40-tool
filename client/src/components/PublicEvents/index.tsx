import PublicEvents from './PublicEvents';

export default PublicEvents;
