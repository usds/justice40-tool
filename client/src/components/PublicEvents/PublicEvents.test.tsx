import * as React from 'react';
import {render} from '@testing-library/react';
import {LocalizedComponent} from '../../test/testHelpers';
import PublicEvents from './PublicEvents';

describe('rendering of the PublicEvents', () => {
  const {asFragment} = render(
      <LocalizedComponent>
        <PublicEvents />
      </LocalizedComponent>,
  );

  it('checks if component renders', () => {
    expect(asFragment()).toMatchSnapshot();
  });
});
