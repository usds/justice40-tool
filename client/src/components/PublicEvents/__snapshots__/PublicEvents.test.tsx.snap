// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering of the PublicEvents checks if component renders 1`] = `
<DocumentFragment>
  <ul
    class="usa-collection"
    data-testid="collection"
  >
    <li
      class="usa-collection__item"
    >
      <img
        alt="Alt text"
        class="usa-collection__img"
        src="test-file-stub"
      />
      <div
        class="usa-collection__body"
      >
        <h3
          class="usa-collection__heading"
        >
          CEJST training session #1
        </h3>
        <p
          class="usa-collection__description"
        >
          
        The White House Council on Environmental Quality (CEQ), in partnership with the U.S. Digital 
        Service, is hosting a series of 'Training Webinars' for users of the Climate and Economic 
        Justice Screening Tool. These webinars are an opportunity for members of the public to learn how to 
        use the current version of the tool. The presenters at these webinars will be available to 
        provide technical support and address issues related to accessing and using the tool.
      
        </p>
        <p
          class="usa-collection__description"
        >
          <b>
            Event info
          </b>
          : March 9th (4:00 - 5:00 PM EST)
        </p>
        <p
          class="usa-collection__description"
        >
          <a
            href="https://pitc.zoomgov.com/webinar/register/WN_D-Om_xXhTtiLv71y3Rr1CQ"
            rel="noreferrer"
            target="_blank"
          >
            <button
              class="usa-button"
              data-testid="button"
              disabled=""
              type="button"
            >
              Registration link
            </button>
          </a>
        </p>
      </div>
    </li>
    <li
      class="usa-collection__item"
    >
      <img
        alt="Alt text"
        class="usa-collection__img"
        src="test-file-stub"
      />
      <div
        class="usa-collection__body"
      >
        <h3
          class="usa-collection__heading"
        >
          CEJST training session #2
        </h3>
        <p
          class="usa-collection__description"
        >
          
        The White House Council on Environmental Quality (CEQ), in partnership with the U.S. Digital 
        Service, is hosting a series of 'Training Webinars' for users of the Climate and Economic 
        Justice Screening Tool. These webinars are an opportunity for members of the public to learn how to 
        use the current version of the tool. The presenters at these webinars will be available to 
        provide technical support and address issues related to accessing and using the tool.
      
        </p>
        <p
          class="usa-collection__description"
        >
          <b>
            Event info
          </b>
          : March 10th (4:00 - 5:00 PM EST)
        </p>
        <p
          class="usa-collection__description"
        >
          <a
            href="https://pitc.zoomgov.com/webinar/register/WN_QsSqshI4TpmRBkI6nVlWxQ"
            rel="noreferrer"
            target="_blank"
          >
            <button
              class="usa-button"
              data-testid="button"
              disabled=""
              type="button"
            >
              Registration link
            </button>
          </a>
        </p>
      </div>
    </li>
    <li
      class="usa-collection__item"
    >
      <img
        alt="Alt text"
        class="usa-collection__img"
        src="test-file-stub"
      />
      <div
        class="usa-collection__body"
      >
        <h3
          class="usa-collection__heading"
        >
          CEJST training session #3
        </h3>
        <p
          class="usa-collection__description"
        >
          
        The White House Council on Environmental Quality (CEQ), in partnership with the U.S. Digital 
        Service, is hosting a series of 'Training Webinars' for users of the Climate and Economic 
        Justice Screening Tool. These webinars are an opportunity for members of the public to learn how to 
        use the current version of the tool. The presenters at these webinars will be available to 
        provide technical support and address issues related to accessing and using the tool.
      
        </p>
        <p
          class="usa-collection__description"
        >
          <b>
            Event info
          </b>
          : March 16th (4:00 - 5:00 PM EST)
        </p>
        <p
          class="usa-collection__description"
        >
          <a
            href="https://pitc.zoomgov.com/webinar/register/WN_q86iMtpwTESYa6f0xpIk7g"
            rel="noreferrer"
            target="_blank"
          >
            <button
              class="usa-button"
              data-testid="button"
              disabled=""
              type="button"
            >
              Registration link
            </button>
          </a>
        </p>
      </div>
    </li>
    <li
      class="usa-collection__item"
    >
      <img
        alt="Alt text"
        class="usa-collection__img"
        src="test-file-stub"
      />
      <div
        class="usa-collection__body"
      >
        <h3
          class="usa-collection__heading"
        >
          CEJST listening session #1
        </h3>
        <p
          class="usa-collection__description"
        >
          
        CEQ is hosting public listening sessions to seek input and feedback on the beta version of the 
        tool, including on the datasets it includes and the methodology it uses. This feedback is critical 
        to the development and enhancement of the tool. This feedback will help CEQ update and refine the 
        tool to ensure that it reflects the environmental, climate and other challenges that communities 
        are experiencing.
      
        </p>
        <p
          class="usa-collection__description"
        >
          <b>
            Event info
          </b>
          : March 22nd (4:00 - 5:00 PM EST)
        </p>
        <p
          class="usa-collection__description"
        >
          <a
            href="https://pitc.zoomgov.com/webinar/register/WN_YT7_uLZqScGHgyAcTCuJjA"
            rel="noreferrer"
            target="_blank"
          >
            <button
              class="usa-button"
              data-testid="button"
              disabled=""
              type="button"
            >
              Registration link
            </button>
          </a>
        </p>
      </div>
    </li>
    <li
      class="usa-collection__item"
    >
      <img
        alt="Alt text"
        class="usa-collection__img"
        src="test-file-stub"
      />
      <div
        class="usa-collection__body"
      >
        <h3
          class="usa-collection__heading"
        >
          CEJST Public WHEJAC meeting day one
        </h3>
        <p
          class="usa-collection__description"
        >
          
        The White House Environmental Justice Advisory Council is also soliciting feedback on the beta version of the Climate and Economic Justice Screening Tool at its public meeting. The link above has additional details.
      
        </p>
        <p
          class="usa-collection__description"
        >
          <b>
            Event info
          </b>
          : March 30th (3:00 - 7:00 PM EST)
        </p>
        <p
          class="usa-collection__description"
        >
          <a
            href="https://usepa.zoomgov.com/webinar/register/WN_wCwVP1dtT0auAR5kfucVtw"
            rel="noreferrer"
            target="_blank"
          >
            <button
              class="usa-button"
              data-testid="button"
              disabled=""
              type="button"
            >
              Registration link
            </button>
          </a>
        </p>
      </div>
    </li>
    <li
      class="usa-collection__item"
    >
      <img
        alt="Alt text"
        class="usa-collection__img"
        src="test-file-stub"
      />
      <div
        class="usa-collection__body"
      >
        <h3
          class="usa-collection__heading"
        >
          <a
            class="usa-link usa-link--external"
            data-cy="mar-31-reg-link-block"
            href="https://usepa.zoomgov.com/webinar/register/WN_wCwVP1dtT0auAR5kfucVtw"
            rel="noreferrer"
            target="_blank"
          >
            CEJST Public WHEJAC meeting day two
          </a>
        </h3>
        <p
          class="usa-collection__description"
        >
          
        The White House Environmental Justice Advisory Council is also soliciting feedback on the beta version of the Climate and Economic Justice Screening Tool at its public meeting. The link above has additional details.
      
        </p>
        <p
          class="usa-collection__description"
        >
          <b>
            Event info
          </b>
          : March 31th (3:00 - 7:30 PM EST)
        </p>
        <p
          class="usa-collection__description"
        >
          <a
            href="https://usepa.zoomgov.com/webinar/register/WN_wCwVP1dtT0auAR5kfucVtw"
            rel="noreferrer"
            target="_blank"
          >
            <button
              class="usa-button"
              data-testid="button"
              type="button"
            >
              Registration link
            </button>
          </a>
        </p>
      </div>
    </li>
    <li
      class="usa-collection__item"
    >
      <img
        alt="Alt text"
        class="usa-collection__img"
        src="test-file-stub"
      />
      <div
        class="usa-collection__body"
      >
        <h3
          class="usa-collection__heading"
        >
          <a
            class="usa-link usa-link--external"
            data-cy="apr-15-reg-link-block"
            href="https://pitc.zoomgov.com/webinar/register/WN_dLw3xChiTlaOLGdHXQWk0w"
            rel="noreferrer"
            target="_blank"
          >
            CEJST listening session #2
          </a>
        </h3>
        <p
          class="usa-collection__description"
        >
          
        CEQ is hosting public listening sessions to seek input and feedback on the beta version of the 
        tool, including on the datasets it includes and the methodology it uses. This feedback is critical 
        to the development and enhancement of the tool. This feedback will help CEQ update and refine the 
        tool to ensure that it reflects the environmental, climate and other challenges that communities 
        are experiencing.
      
        </p>
        <p
          class="usa-collection__description"
        >
          <b>
            Event info
          </b>
          : April 15th (4:00 - 5:00 PM EST)
        </p>
        <p
          class="usa-collection__description"
        >
          <a
            href="https://pitc.zoomgov.com/webinar/register/WN_dLw3xChiTlaOLGdHXQWk0w"
            rel="noreferrer"
            target="_blank"
          >
            <button
              class="usa-button"
              data-testid="button"
              type="button"
            >
              Registration link
            </button>
          </a>
        </p>
      </div>
    </li>
  </ul>
</DocumentFragment>
`;
